test/plugin/scenarios/armeria-scenario/bin/startup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -jar ${agent_opts} -Dskywalking.agent.service_name=armeria-server ${home}/../libs/armeria-scenario.jar &
```

Comment:
Hi, `agent_opts` already contains `agent.service_name`, which is not normally set here.

https://github.com/apache/skywalking/blob/34fd8637c05ca9a8de07bc90080d3c54eee5f02c/test/plugin/containers/jvm-container/docker/run.sh#L65-L73

Reply:
Settings are usually required when an case has more than one service.
https://github.com/apache/skywalking/blob/34fd8637c05ca9a8de07bc90080d3c54eee5f02c/test/plugin/scenarios/webflux-scenario/webflux-dist/bin/startup.sh#L19-L24

Reply:
> Hi, `agent_opts` already contains `agent.service_name`, which is not normally set here.

Hum, seems you yourself did this too, anyway, resolved

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/15965696/70385984-6237c180-19ce-11ea-89dc-96f4792dd676.png">